.github/workflows/e2e-tests.yml

```
@@ -0,0 +1,221 @@
+        run: |
+          pnpm --filter @bluelight-hub/frontend exec playwright test \
+            --grep "Visual Regression" \
+            --update-snapshots
+
+      - name: Upload visual diff
+        if: failure()
+        uses: actions/upload-artifact@v3
+        with:
+          name: visual-regression-diff
+          path: packages/frontend/test-results/**/*-diff.png
+          retention-days: 7
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)